jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/SortingOverview.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+
```

Comment:
There are so much redundant empty lines in PR.
Please review all classes and remove them.

Reply:
Fixed